### DISCUSSION TITLE
Welcome to GeoRaptor Discussions!

### DISCUSSION THREAD

**Discussion by sggreener** [sggreener/GeoRaptor#13] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by JanisZageris** [comment]
Hello, I was hoping you could help me with the install, as currently trying to install it on my sql developer (Version 23.1.1.345; Build 345.2114) I've gotten an error saying that META-INF doesn't contain bundle.xml. After moving it from -master to META-INF, it is saying that the bundle.xml contains errors. I've also noticed that there is an option to get the files from spdba.com, however there I am unable to make the purchase due to an unknown error. Assistance would be most appreciated.

**Comment by sggreener** [comment]
I've never built from github directly except to use eclipse. The code should be up to date. I'll send you a link to an install zip for free. 

⁣Simon
www.spdba.com.au

39 Cliff View Drive, Allens Rivulet, 7150, Tasmania, Australia

(P) 03 62 396 397
(M) 0418 396 391

O good old man, how well in thee appears
The constant service of the antique world,
When service sweat for duty, not for meed.
Thou art not for the fashion of these times,
Where none will sweat but for promotion.
Shakespeare, As you like it.​

**Comment by sggreener** [comment]
https://www.dropbox.com/scl/fi/ew1ubum4p78nnni6uioe8/GeoRaptor.zip?rlkey=diy73kfrvcuyq2vdan2e8oit8&st=9e69yofd&dl=0


⁣Simon
www.spdba.com.au

39 Cliff View Drive, Allens Rivulet, 7150, Tasmania, Australia

(P) 03 62 396 397
(M) 0418 396 391

O good old man, how well in thee appears
The constant service of the antique world,
When service sweat for duty, not for meed.
Thou art not for the fashion of these times,
Where none will sweat but for promotion.
Shakespeare, As you like it.​

**Reply by JanisZageris** [reply]
Amazing, thank you for sharing the install!
